odoo/models.py

```diff
@@ -156,31 +156,18 @@ def __init__(self, name, bases, attrs):
             return
 
         if not hasattr(self, '_module'):
-            self._module = self._get_addon_name(self.__module__)
+            assert self.__module__.startswith('odoo.addons.')
+            self._module = self.__module__.split('.')[2]
 
         # Remember which models to instanciate for this module.
         if not self._custom:
             self.module_to_models[self._module].append(self)
 
         # check for new-api conversion error: leave comma after field definition
         for key, val in attrs.items():
-            if type(val) is tuple and len(val) == 1 and isinstance(val[0], Field):
-                _logger.error("Trailing comma after field definition: %s.%s", self, key)
```

Comment:
Why remove this lint-ish sanity check?

Reply:
it is no more relevant according to rco, I should remove the comment btw